recipes/Florentiner.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "The best Florentiners"
+image: "https://heissehimbeeren.com/wp-content/uploads/2022/11/omas-florentiner-rezept-12.webp"
+image-alt: "https://images.lecker.de/klassische-florentiner/1x1,id=72f9c685,b=lecker,w=1600,h=,ca=12.50,0,87.50,100,rm=sk.jpeg"
+categories:
+  - Christmas
+  - Cookie
+  - Sweet
+  - Vegetarian
```

Comment:
Thank you @martinaengel for these additional changes! Would you you like to add your author name?
For this add your name to the author field in the YAMl header, like this:

```YAML
author: firstname lastname
```